orchagent/port/porthlpr.cpp

```diff
@@ -667,6 +667,7 @@ bool PortHelper::parsePortLinkTraining(PortConfig &port, const std::string &fiel
     return true;
 }
 
+
```

Comment:
Can you please revert the changes that are not part of this PR? like this extra line?

Reply:
We removed the extra lines that were added during the coding process.